Healthy/Classes/Modules/Login/LoginViewModel.swift

```diff
@@ -1,21 +1,57 @@
 import Foundation
+import Combine
 
 final class LoginViewModel {
-
+    private var store = Set<AnyCancellable>()
```

Comment:
```suggestion
    private var subscriptions = Set<AnyCancellable>()
```

Consistency is the 🔑